docs/develop/api-reference/dictionary-collections.md

```diff
@@ -0,0 +1,201 @@
+---
+sidebar_position: 1
+sidebar_label: Dictionary collections
+title: Dictionary collection API reference
+description: Learn how to interact with the API for dictionary collection data types in Momento Serverless Cache.
+slug: /develop/api-reference/collections/dictionary
+---
+
+# Dictionary collection API reference
+
+## Dictionary methods
+This page details the Momento API methods for [dictionary collections](./../datatypes.md#dictionary-collections).
```

Comment:
You asked about this in slack; in the context here I think this reads fine, personally.